ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -592,6 +592,32 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		}
 	}
 
+	func upload(asset:PHAsset) {
+		let progress = Progress(totalUnitCount: 100)
```

Comment:
Please consider adding a `localizedDescription`, so users who get to see this progress know what action it is for. F.ex. "Copying [name of asset] from photo library…"

Reply:
@felix-schwarz added